### ISSUE TITLE
"variable might be clobbered by ‘longjmp’ or ‘vfork’" warning (error) in RouteGen_admin.c

### ISSUE THREAD

**Issue by laanwj** [hyperboria/cjdns#125]
Not sure how serious this warning is, but due to Werror this will not let me proceed:

gcc `(Debian 4.9.2-10) 4.9.2` on ARM 32-bit

```
Error: gcc -c -x cpp-output -o build_linux/tunnel_RouteGen_admin_c.o -std=c99 -Wall -Wextra -Werror -Wno-pointer-sign -pedantic -D linux=1 -D CJD_PACKAGE_VERSION="cjdns-v17.3-106-gff94aa6" -Wno-unused-parameter -D Log_DEBUG -g -D NumberCompress_TYPE=v3x5x8 -D Identity_CHECK=1 -D Allocator_USE_CANARIES=1 -D PARANOIA=1 -DHAS_ETH_INTERFACE=1 -fPIE -fno-stack-protector -fstack-protector-all -Wstack-protector -O3 build_linux/tunnel_RouteGen_admin_c.o.i

tunnel/RouteGen_admin.c: In function ‘commit’:
tunnel/RouteGen_admin.c:181:11: error: variable ‘error’ might be clobbered by ‘longjmp’ or ‘vfork’ [-Werror=clobbered]
     char* error = "none";
           ^
cc1: all warnings being treated as errors
```

**Comment by ghost** [comment]
Try the fix in cjdelisle/cjdns#922

**Comment by ghost** [comment]
It's merged into cjd's crashey branch now
